stack.py

```diff
@@ -68,7 +68,8 @@ def _process_stack_cfg(cfg, stack, minion_id, pillar):
             jenv.get_template(filename).render(stack=stack)):
         if not item.strip():
             continue  # silently ignore whitespace or empty lines
-        paths = glob.glob(os.path.join(basedir, item))
+        # allow indentation before lines in stack config
+        paths = glob.glob(os.path.join(basedir, item.lstrip()))
```

Comment:
whitespace should have already been removed with the `item.strip()` instruction above, so `lstrip` is not needed here.

Reply:
`item.strip()` returns a new string, leaving `item` unmodified, so I think this is still necessary if you want to allow indentation.

Reply:
you are right, so we should do `item = item.strip()`